### ISSUE TITLE
[SUPPORT] 

### ISSUE THREAD

**Issue by radoeka** [pbek/QOwnNotes#2816]
#### Question

I've no more settings, help, about or similar menu.
Because of this, I'm for example not able to change any settings or obtain information about QOwnNotes.

What to do to obtain the settings, help, about or similar menu back?

#### Output from the debug section in the settings dialog

<details><summary>Expand</summary>

As the menu's are not available to me, I can't provide any information :(

Some screenshots to show what I currently have:
  
![image](https://github.com/pbek/QOwnNotes/assets/2937666/f85b7dcf-190d-4d3b-91c9-678d4947cc69)

![image](https://github.com/pbek/QOwnNotes/assets/2937666/257820ac-fe7e-4498-b80c-7e967ba0e08f)
  (some icons are shown twice)

Is it possible to launch QOwnNotes with options, so it starts with the configuration or welcome dialog?

This is on MS Windows.

QOwnNotes 23.8.1, but shows the same on 23.4.x

</details>

#### Relevant log output in the Log panel

<details><summary>Expand</summary>

</details>

Hopefully, this can be fixed.

**Comment by pbek** [comment]
I'm not sure if I understand the question correctly.
If you want to try different settings, there is a command line options to start a different settings session to try out new settings:

https://www.qownnotes.org/getting-started/cli-parameters.html

**Comment by pbek** [comment]
You can also reset the settings of your default session and start from scratch.

**Comment by radoeka** [comment]
> https://www.qownnotes.org/getting-started/cli-parameters.html
@pbek : this helped a lot.  With that information I was able to start a new session.
That showed that I did not have the menu bar, and it showed how to hide it &lt;shift>&lt;ctrl>&lt;m>.

That key combination enables it as well.  So, that's what I used.

![image](https://github.com/pbek/QOwnNotes/assets/2937666/7612e80d-f5c1-4088-afa4-2e456ed7a139)

My issue is solved :)

But, it it might be useful to add a Note somewhere to tell people how to get the menubalk back...
Perhaps a note in the --help text?
